microsoft/vscode
# Add property default show context value proeprty

This PR fixes (implements?) #97915

Add the `search.searchEditor.defaultShowContextValue` settings option to configure the default value for the number of context lines.

To test add
```
"search.searchEditor.defaultShowContextValue": 2
```

to settings, a new search editor should now show 2 lines of context.

`reusePriorSearchConfiguration` should take precedence though, so with settings like
```
"search.searchEditor.reusePriorSearchConfiguration": true,
"search.searchEditor.defaultShowContextValue": 2
```

new search editor windows should show the previously set number of context lines.
